CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## 1.13.0 (April 10, 2024)
+
+ * [#4045](https://github.com/redhat-developer/vscode-openshift-tools/issues/4045) Can't start crc via the extension.
+ * [#4013](https://github.com/redhat-developer/vscode-openshift-tools/issues/4013) Provide an option to delete the Pod like k8s.
+ * [#4004](https://github.com/redhat-developer/vscode-openshift-tools/issues/4004) Survey doesn't work.
+ * [#3991](https://github.com/redhat-developer/vscode-openshift-tools/issues/3991) Re-arrange application explorer similar to vsKubernetes cluster view.
+ * [#3981](https://github.com/redhat-developer/vscode-openshift-tools/issues/3981) Migrate Functionality from Kubernetes VSCode Extension to OpenShift Toolkit.
+ * [#3947](https://github.com/redhat-developer/vscode-openshift-tools/issues/3947) The App Explorer tree isn't refreshed after logging in to a Kind cluster.
+ * [#3919](https://github.com/redhat-developer/vscode-openshift-tools/issues/3919) Improve "Open Kuberbetes YAML file" to avoid accidental opens and duplicate files.
+ * [#3918](https://github.com/redhat-developer/vscode-openshift-tools/issues/3918) Fix Action labels for kubernetes clusters.
```

Comment:
I think this should be more specific, eg. "Use "namespaces" wording on kubernetes clusters and "projects" wording on OpenShift clusters"

Reply:
ACK